collectors/0/flume_offset_collector.py

```diff
@@ -6,7 +6,7 @@
 class FlumeOffsetCollector(KafkaOffsetCollector):
     KAFKA_CHROOT = ""
     ZK_QUORUM = ["zookeeperKafkaProd-1", "zookeeperKafkaProd-2", "zookeeperKafkaProd-3", "zookeeperKafkaProd-4", "zookeeperKafkaProd-5"]
-    CONSUMER_GROUPS = {"flumeS3", "flumeHDFS", "flumeP13N"}
+    CONSUMER_GROUPS = {"flumeS3", "flumeHDFS", "flumeP13N", "flumeLokiAchievement", "flumeLokiFlatSegmentValue", "flumeStagingAchievement", "flumeStagingFlatSegmentValue"}
```

Comment:
This is fine, but you might want to consider either black listing known consumers to ignore, or using a pattern match on consumer groups we want to monitor (e.g. start with "flume").